sw/host/opentitantool/src/command/image.rs

```diff
@@ -223,7 +231,9 @@ impl CommandDispatch for ManifestUpdateCommand {
         }
 
         // Update `length` to be able to generate signatures.
-        image.update_length()?;
```

Comment:
does this conflict with @jon-flatley's recent PR?

Reply:
Probably.  I'll wait for Jon's PR to submit and then update mine accordingly.

Reply:
I've rebased now that #18893 is in and ran a couple of tests locally.  Re-pushed to run everything through CI.